Rakefile

```diff
@@ -1,17 +1,20 @@
-# frozen_string_literal: true
-
 require 'English'
+# Constants
+SWIFTLINT_VERSION = '0.41.0'
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Automattic/simplenote-ios/blob/e04cb11e59083cd6d00575514a0508527381d352/Rakefile#L3"></span>Style/MutableConstant: Freeze mutable objects assigned to constants.</td>
    </tr>
  </tbody>
</table>